Assets/MixedRealityToolkit.Tools/MSBuild/MSBuildTemplates/SDKProjectTemplate.csproj.template

```diff
@@ -17,6 +17,7 @@
     <RelativeBuildFolder>..\Output\</RelativeBuildFolder>
     <BuildFolder>$(MSBuildThisFileDirectory)$(RelativeBuildFolder)</BuildFolder>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+	<Deterministic>false</Deterministic>
```

Comment:
><Deterministic>false</Deterministic> [](start = 1, length = 36)

Do you have some more details to share on why we're moving to non-deterministic builds (i.e. when deterministic appears to be the default and the direction that things are moving in?)

Even having some like, maybe some comments inline here as well would be helpful to future readers

Reply:
One of the packages has an AssemblyInfo that has [assembly: AssemblyVersion("X.Y.*")] where X and Y are numbers, but the * fails the build if it is deterministic

---
In reply to: [310749157](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5516#discussion_r310749157) [](ancestors = 310749157)